tests/unit/backend/recipe/algorithm/test_PurgeOverlappingPeaksAlgorithm.py

```diff
@@ -22,35 +21,26 @@ def test_init():
         instrumentState = Calibration.parse_raw(
             Resource.read("/inputs/purge_peaks/input_parameters.json")
         ).instrumentState
-        focusGroups = ReductionIngredients.parse_raw(
-            Resource.read("/inputs/purge_peaks/input_ingredients.json")
-        ).reductionState.stateConfig.focusGroups
         crystalInfo = CrystallographicInfo.parse_raw(Resource.read("/inputs/purge_peaks/input_crystalInfo.json"))
         purgeAlgo = PurgeOverlappingPeaksAlgorithm()
         purgeAlgo.initialize()
         purgeAlgo.setProperty("InstrumentState", instrumentState.json())
-        purgeAlgo.setProperty("FocusGroups", json.dumps([focusGroup.json() for focusGroup in focusGroups]))
         purgeAlgo.setProperty("CrystalInfo", crystalInfo.json())
         assert purgeAlgo.getProperty("InstrumentState").value == instrumentState.json()
-        assert purgeAlgo.getProperty("FocusGroups").value == json.dumps(
```

Comment:
Can we assert the number of groups is what it should be here?

Reply:
This has been removed from the algorithm, it no longer uses Focus Groups